Makefile

```diff
@@ -13,7 +13,7 @@ upgrade:
 	pip install -e . -U
 
 lint:
-	flake8
+	flake8 --exclude snap,parts,stage,prime
```

Comment:
can go in setup.cfg the exclude stuff, I thought we already had some there

Reply:
But `setup.cfg` doesn't get used if I just run `make lint` on the command line during development.

Reply:
I'll have a look when I get back in November but yes the flake8 tool does read setup.cfg for config if there is a [flake8] section. If we are to publish snaps using travis it has to be on merge, but to be honest I am not a huge fan of snaps and didn't really want to get involved in maintenance of snaps but I'll have a look in early November. Problem with the project is I'm the only person still merging stuff for the last 2 years and I'm insanely busy sometimes it takes a while as I like to test everything properly.